jetty-ee9/jetty-ee9-quickstart/src/main/java/org/eclipse/jetty/ee9/quickstart/QuickStartConfiguration.java

```diff
@@ -213,52 +215,80 @@ protected void quickStart(WebAppContext context)
         _quickStart = true;
         context.setConfigurations(context.getConfigurations().stream()
             .filter(c -> !__replacedConfigurations.contains(c.replaces()) && !__replacedConfigurations.contains(c.getClass()))
-            .collect(Collectors.toList()).toArray(new Configuration[]{}));
-        context.getMetaData().setWebDescriptor(new WebDescriptor((Resource)context.getAttribute(QUICKSTART_WEB_XML)));
+            .toList()
+            .toArray(new Configuration[]{}));
```

Comment:
Same comments as ee10's version of this file.

Reply:
Done.